packages/help-center/src/components/help-center-content.tsx

```diff
@@ -131,12 +131,14 @@ const HelpCenterContent: React.FC< { isRelative?: boolean; currentRoute?: string
 								sectionName={ sectionName }
 								currentSiteDomain={ currentSiteDomain }
 								isEligibleForChat={ isUserEligibleForPaidSupport }
-								forceEmailSupport={ !! forceEmailSupport }
+								forceEmailSupport={ !! forceEmailSupport || disableChatSupport }
```

Comment:
If chat is disabled in the Help center level, we force email support in the support article.